packages/actor-rdf-resolve-quad-pattern-hypermedia/lib/MediatedQuadSource.ts

```diff
@@ -43,8 +43,13 @@ export class MediatedQuadSource implements IQuadSource {
     predicate: RDF.Term,
     object: RDF.Term,
     graph: RDF.Term,
-    context: IActionContext,
+    context?: IActionContext,
   ): AsyncIterator<RDF.Quad> {
+    // IQuadSource["context"] should be mandatory in the next major update.
+    if (!context) {
+      throw new Error(`MediatedQuadSource requires a context.`);
```

Comment:
This change will probably require a new unit test to achieve 100% test coverage.